src/htdocs/css/_list.scss

```diff
@@ -20,6 +20,11 @@ ol {
 
 .linklist {
 
+ul,
```

Comment:
I think you want:
```
.linklist {
  padding: 0;

  > li
  ...
```

The `.linklist` class is on the `ul`, `ol` elements. So this style won't target the padding. Also, generally when we setting a css property to zero we just use "0", no units needed.